python/py-botocore/Portfile

```diff
@@ -18,20 +18,29 @@ long_description    A low-level interface to a growing number of Amazon Web \
 
 homepage            https://github.com/boto/botocore
 
-checksums           rmd160  ab01b00e2f2c023db1a0c2b9309f8261a4b13dd6 \
-                    sha256  e4132d34e06356b7bb14838050360025700fda10294550807779154d303eaf9f \
-                    size    5653940
+checksums           rmd160  26240e76c5f0455410537d11c76d5c50f9869f38 \
+                    sha256  ee55ce128056c5120680d25c8e8dfa3a08dbe7ac3445dc16997daaa68ae4060e \
+                    size    5789267
 
-python.versions     27 35 36 37 38
+python.versions     27 37 38
 
 if {${name} ne ${subport}} {
-    depends_build-append    port:py${python.version}-setuptools
-    depends_lib-append      port:py${python.version}-dateutil \
-                            port:py${python.version}-docutils \
-                            port:py${python.version}-jmespath \
-                            port:py${python.version}-urllib3
+    depends_build-append \
+                    port:py${python.version}-setuptools
+    depends_lib-append \
+                    port:py${python.version}-dateutil \
+                    port:py${python.version}-docutils \
+                    port:py${python.version}-jmespath \
+                    port:py${python.version}-jsonschema \
+                    port:py${python.version}-mock \
+                    port:py${python.version}-nose \
+                    port:py${python.version}-tox \
```

Comment:
I find it hard to believe that thing like mock, nose, and tox are anything more than test dependencies

Reply:
I know but that's what the setup.py says…

Reply:
I have to disagree, none of these packages are even mentioned in ```setup.py```...

Reply:
Correction: requirements.txt:
https://github.com/boto/botocore/blob/be2bd7ef80699d2c9e94ce7ed0cf69ea7e76b3fe/requirements.txt#L1-L7

Reply:
well, but that doesn't say anything about *what* type of dependency it is, and they are not necessarily the minimum set of dependencies that are needed? 

https://packaging.python.org/discussions/install-requires-vs-requirements/

Reply:
Upstream imports them in tests, e.g. https://github.com/boto/botocore/search?q=jsonschema&unscoped_q=jsonschema

I'll just toss them into depends_test.